### PR TITLE
fixed bug of invalid expression expression caused graph crash

### PR DESCRIPTION
**Pull request by bright-starry-sky**
For NG-383, I can't repeat the issue. so added some test cases for regression testing.

### CODE REVIEW

**Review by yixinglu — APPROVED** [review]

**Review by Shylock-Hg — APPROVED** [review]

**Comment by bright-starry-sky** [comment]
Sorry, please hold this PR, I need recheck it.

**Comment by bright-starry-sky** [comment]
fixed done. ready to review again . thanks.

**Review by yixinglu — APPROVED** [review]

File: src/validator/LookupValidator.cpp [hunk 1]
```diff
@@ -388,6 +388,12 @@ Status LookupValidator::rewriteRelExpr(RelationalExpression* expr) {
 }
 
 StatusOr<Value> LookupValidator::checkConstExpr(Expression* expr, const std::string& prop) {
+    // TODO (sky) : just check the const expression now.
+    // other expressions will be supported in the future. such as :
+    // col1 > 1 + 2 , col1 > abs(const) , etc.
+    if (expr->kind() != Expression::Kind::kConstant) {
+        return Status::SemanticError("expression error : %s", expr->toString().c_str());
+    }
```
Comment (CPWstatic) [inline src/validator/LookupValidator.cpp:396]: we could use `evaluableExpr` here.

Comment (bright-starry-sky) [reply src/validator/LookupValidator.cpp]: > we could use `evaluableExpr` here.

Thanks, this is what I really want. fixed.

**Review by CPWstatic — APPROVED** [review]

**Review by yixinglu — APPROVED** [review]

**Pull request merged**